sokol_app.h

```diff
@@ -3857,6 +3861,16 @@ _SOKOL_PRIVATE bool _sapp_win32_utf8_to_wide(const char* src, wchar_t* dst, int
     }
 }
 
+_SOKOL_PRIVATE void _sapp_win32_show_mouse(bool shown) {
+    ShowCursor((BOOL)shown);
+}
+
+_SOKOL_PRIVATE bool _sapp_win32_mouse_shown(void) {
+    CURSORINFO cursor_info = { .cbSize = sizeof(CURSORINFO) };
```

Comment:
Please no C99 in the implementation code, must also compile as C++ :)

Reply:
PS: since something like ```CURSORINFO cursor_info = {0};``` also creates warnings on some compilers, the recommended way to clear out a struct in the sokol implementations is a good old memset().